make distclean fails with sleuthkit-4.14.0 due to missing ant

`make distclean` fails due to missing ant, tested with https://github.com/sleuthkit/sleuthkit/releases/tag/sleuthkit-4.14.0

```
make distclean

...
make[2]: Entering directory '/home/user/Projects/sleuthkit/bindings/java'
test -z "../../bindings/java/dist/sleuthkit-4.14.0.jar" || rm -f ../../bindings/java/dist/sleuthkit-4.14.0.jar
rm -rf .libs _libs
ant clean
make[2]: ant: No such file or directory
make[2]: *** [Makefile:741: clean-local] Error 127
make[2]: Leaving directory '/home/user/Projects/sleuthkit/bindings/java'
make[1]: *** [Makefile:462: distclean-recursive] Error 1
make[1]: Leaving directory '/home/user/Projects/sleuthkit/bindings/java'
make: *** [Makefile:588: distclean-recursive] Error 1
```

--- a/bindings/java/Makefile.am
+++ b/bindings/java/Makefile.am
@@ -20,4 +20,4 @@ all-local:
 CLEANFILES = $(tsk_jar)
 
 clean-local:
-	ant clean
+	ant clean || echo ant not installed

--- a/case-uco/java/Makefile.am
+++ b/case-uco/java/Makefile.am
@@ -17,4 +17,4 @@ all-local:
 CLEANFILES = $(tsk_caseuco_jar)
 
 clean-local:
-	ant clean
+	ant clean || echo ant not installed

